lib/rabbit_client/bunny_client.rb

```diff
@@ -10,6 +10,34 @@ def self.publish_messages(url, exchange_name, exchange_type, messages)
       end
     end
 
+    def self.publish_message(url, exchange_name, exchange_type, message, routing_key, batch, origin)
+      connect url, exchange_name, exchange_type do |exchange|
+
+        current_time = Time.now.getutc
+        organization = message.organization
+
+        message_headers = {}
+
+        message_headers[:routing_key] = routing_key
+        message_headers[:organization] = organization
+        message_headers[:batch] = batch
```

Comment:
We need to discuss what we mean with 'batch', and what implications it should have